tensorflow_model_optimization/g3doc/guide/roadmap.md

```diff
@@ -9,9 +9,9 @@
 *   Quantization aware training for (8b) fixed-point kernels and experimentation
     for <8b --
     [Launched](https://blog.tensorflow.org/2020/04/quantization-aware-training-with-tensorflow-model-optimization-toolkit.html)
-*   Post training quantization for (8b) fixed-point RNNs
+*   [WIP] Post training quantization for (8b) fixed-point RNNs
```

Comment:
@raziel: should this be changed to launched? Didn't see it being mentioned in the blog post on fused LSTMs, so I left it as WIP for now.